README.md

```diff
@@ -65,19 +72,19 @@ Running the Examples
 
 Load data points from a file:
 ```
-p2t <filename> <center_x> <center_y> <zoom>
+build/testbed/p2t <filename> <center_x> <center_y> <zoom>
```

Comment:
Isn't `./` required for most shells?

Reply:
Ah, makes sense! Thanks :)

I never thought about the / being the requirement, not the dot.